examples/logical_interconnect.rb

```diff
@@ -62,8 +64,19 @@
 oneview_logical_interconnect 'Encl1-LogicalInterconnectGroup1' do
   client my_client
   data(
-    'portMonitor' => {
-      'enabled' => true
+    portMonitor: {
+      analyzerPort: {
+        portUri: '/rest/interconnects/c3498956-ece5-4abf-9e7f-793cee92acca/ports/c3498956-ece5-4abf-9e7f-793cee92acca:1',
```

Comment:
Just a question, the value of portUri is variable or is previsible? How will client set this value at runtime of recipes?

Reply:
It's ok! I'll implement a way to get the uri by the port name. What do you think?

Reply:
Nice! =)